RoddyCore/src/de/dkfz/roddy/execution/io/LocalExecutionService.groovy

```diff
@@ -10,16 +10,30 @@ import de.dkfz.roddy.SystemProperties
 import de.dkfz.roddy.config.Configuration
 import de.dkfz.roddy.config.ConfigurationConstants
 import de.dkfz.roddy.config.ConfigurationValue
+import de.dkfz.roddy.execution.UnexpectedExecutionResultException
 import de.dkfz.roddy.tools.LoggerWrapper
+import groovy.transform.CompileStatic
+
+import java.util.concurrent.Callable
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import java.util.concurrent.Future
+import java.util.function.Consumer
+import java.util.function.Supplier
 
 /**
  * The local execution service executes commands on the local machine. For this groovy's execute() method is used.
- * @author michael
+ * All commands are executed using Bash as interpreter!
```

Comment:
This was not the case before. The change corrects the inconsistent behaviour.

Reply:
What is your opinion?

Reply:
See the comparison of `String.execute()` and `bash -c` in a `ProcessBuilder`, below.

Reply:
Not sure what the thumb-up means. "Good discussion"? "Leave it like it is"? "Poor guy. Keep up the good mood!" ;-)